### ISSUE TITLE
Recent files wrong thumbnail or link to file ?

### ISSUE THREAD

**Issue by XDelta778** [SimpleMobileTools/Simple-File-Manager#703]
Hi, I have an issue using the recent file tab on simple file manager.
When I hit a thumbnail of a picture for example, it shows me another picture. This for every image in this tab.

Is this a known issue ? Can it be fixed ?

**Comment by tibbi** [comment]
it is only about the recents tab? Opening the file from the normal place works well?

**Comment by XDelta778** [comment]
Not it's not working. I opened a file in its place, it worked. But when I try to open any other file in this folder, It open the first file.


⁣Télécharger BlueMail pour Android ​

Le 6 mai 2023, 22:13, à 22:13, Tibor Kaputa ***@***.***> a écrit:

**Comment by tibbi** [comment]
are you using Simple Gallery or some other one?

**Comment by XDelta778** [comment]
I do not have simple gallery, I think it uses m'y default gallery app (Xiaomi) 

⁣Télécharger BlueMail pour Android ​

Le 7 mai 2023, 09:57, à 09:57, Tibor Kaputa ***@***.***> a écrit:

**Comment by tibbi** [comment]
so to confirm, the proper image isnt open even from casually navigating to some folder at the first tab? Does any other file manager + gallery work well?

**Comment by XDelta778** [comment]
Yes exactly. But On other folders with less files, it works well.
On another  file manager it works well. 

I just tried using simple gallery and it work great ! Maybe the issue comes from Xiaomi gallery app.

⁣Télécharger BlueMail pour Android ​

Le 7 mai 2023, 11:21, à 11:21, Tibor Kaputa ***@***.***> a écrit:

**Comment by tibbi** [comment]
what file manager app version are you using at all?

**Comment by XDelta778** [comment]
It is the version 6.15.2 of the simple file manager 

⁣Télécharger BlueMail pour Android ​

Le 7 mai 2023, 11:58, à 11:58, Tibor Kaputa ***@***.***> a écrit:
